Content.Shared/Glue/GlueComponent.cs

```diff
@@ -1,15 +1,42 @@
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
 using Robust.Shared.Audio;
 using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 
-namespace Content.Shared.Glue
+namespace Content.Shared.Glue;
+
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedGlueSystem))]
+public sealed class GlueComponent : Component
 {
-    [RegisterComponent, NetworkedComponent]
-    public sealed class GlueComponent : Component
-    {
-        /// <summary>
-        /// Noise made when glue applied.
-        /// </summary>
-        [DataField("squeeze")]
-        public SoundSpecifier Squeeze = new SoundPathSpecifier("/Audio/Items/squeezebottle.ogg");
-    }
+    /// <summary>
+    /// Noise made when glue applied.
+    /// </summary>
+    [DataField("squeeze")]
+    public SoundSpecifier Squeeze = new SoundPathSpecifier("/Audio/Items/squeezebottle.ogg");
+
+    /// <summary>
+    /// Solution on the entity that contains the glue.
+    /// </summary>
+    [DataField("solution")]
+    public string Solution = "drink";
+
+    /// <summary>
+    /// Reagent that will be used as glue.
+    /// </summary>
+    [DataField("reagent", customTypeSerializer: typeof(PrototypeIdSerializer<ReagentPrototype>))]
+    public string Reagent = "SpaceGlue";
+
+    /// <summary>
+    /// Reagent consumption per use.
+    /// </summary>
+    [DataField("consumption"), ViewVariables(VVAccess.ReadWrite)]
+    public FixedPoint2 Consumption = FixedPoint2.New(3);
+
+    /// <summary>
+    /// Duration per unit
+    /// </summary>
+    [DataField("duration"), ViewVariables(VVAccess.ReadWrite)]
+    public TimeSpan Duration = TimeSpan.FromSeconds(10);
```

Comment:
Rename to DurationPerUnit, add another datafield capping it to like 60 seconds (so you cant put 7 bottles of glue on something for a de-facto admin smite)  (maybe make it not possible to pour more than the cap and do a popup to have some player feedback)

Reply:
right now you can't put glue on item more than once so it doesn't stack